test/tests/data/search/a.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>hello</title>
+  </head>
+  <body>
+    <p>mjktkiuewf</p>
```

Comment:
Yes, sorry if I wasn't clear — the strings should remain as they were before, so that they're easily readable, and we'll deal with DB resetting, which isn't an issue now, separately. (There may be dozens more tests, so having to parse the difference between random strings wouldn't be fun.)